src/css/plate-layout.css

```diff
@@ -304,6 +304,7 @@
 
 .plate-setup-tab-check-box {
   margin-top: 26px;
```

Comment:
I'm ok with using glyphs instead of images but the css here needs to look better. Checkbox needs to be defined with a border, square, centered checkmark, aligned with top of field (not the field title): 

```
.plate-setup-tab-field-left-side {
  ...
padding-top: 16px;
}
.plate-setup-tab-name {
...
}
line-height: 16px;
}
.plate-setup-tab-check-box {
cursor: pointer; 
border: 1px solid gray; //Or similar
display: inline-block;
height: 16px;
width: 16px;
text-align: center; 
line-height: 16px;
}
```

Reply:
AFAICT this issue is not part of the change, may I suggest addressing it in a different pull request ?

Reply:
Ok so that's because he replaced images by glyphs.